disnake/embeds.py

```diff
@@ -104,7 +104,11 @@ class _EmbedAuthorProxy(Protocol):
         proxy_icon_url: MaybeEmpty[str]
 
 
-class Embed:
+class EmbedMetaclass(type):
+    pass
```

Comment:
Why do we need an empty metaclass?

Reply:
iirc it's required for the `setattr(type(owner), name, self)` in `alias_of.__set_name__`, since `type(owner)` would otherwise be `type` and you can't `setattr()` on builtins

Reply:
Completely rewrote this implementation, now using class methods, to avoid class descriptors and metaclasses altogether.